tools/manifest-templator/manifest-templator.go

```diff
@@ -276,6 +282,8 @@ func main() {
 					},
 				})
 			}
+
+			util.AddEnvAcrossContainers(strategySpec, hcoKvIoVersionName, *hcoKvIoVersion)
```

Comment:
I'd prefer to respect the deployment section we get from other operators and avoid directly manipulating it here.
On my opinion it will be sefer to patch the csv-generator tool on all the operators instead of manipulating it here.